src/cr2image.cpp

```diff
@@ -109,10 +109,7 @@ WriteMethod Cr2Parser::encode(BasicIo& io, const byte* pData, size_t size, ByteO
   ExifData ed = exifData;
 
   // Delete IFDs which do not occur in TIFF images
-  static constexpr auto filteredIfds = std::array{
-      IfdId::panaRawId,
-  };
-  for (auto&& filteredIfd : filteredIfds) {
+  for (auto&& filteredIfd : {IfdId::panaRawId}) {
```

Comment:
I'd maybe leave all of these 1-element arrays as is, how can we be sure we won't be adding more elements in the future?

Reply:
Will do